ci/prepare.yml

```diff
@@ -4,7 +4,20 @@
     - name: Install pre-reqs from pip
       ansible.builtin.pip:
         name:
+          - openshift
           - kubernetes
+          - ansible-core>=1.12
```

Comment:
ansible and ansible-core have different versions; I will double check that, but I think this is right for ansible-core.

Reply:
I feel like I've asked this before and was told it was right :) Just that if I run `ansible --version` I get this:

```
ansible [core 2.12.10]
  config file = /etc/ansible/ansible.cfg
  configured module search path = ['/home/leif/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/lib/python3.10/site-packages/ansible
  ansible collection location = /home/leif/.ansible/collections:/usr/share/ansible/collections
  executable location = /usr/bin/ansible
  python version = 3.10.11 (main, Apr  5 2023, 00:00:00) [GCC 12.2.1 20221121 (Red Hat 12.2.1-4)]
  jinja version = 3.0.3
  libyaml = True
```

I get this in my `pip list` as well:

```
pip list | grep ansible
ansible-core                 2.12.10
ansible-lint                 6.14.2
```

I guess it says here >= 1.12, so I suppose that means it could result in 2.x series as well? Just looks really odd to me.